modules/document_repository/jsx/deleteCategoryForm.js

```diff
@@ -106,9 +105,11 @@ class DeleteDocCategoryForm extends React.Component {
     );
   }
 
-  /** *******************************************************************************
+  /**
+   * *******************************************************************************
```

Comment:
?

Reply:
@driusan I think this is an autoformatting change, it did the same thing when I used `--fix` for the indent linting PR (though I reverted it manually). So this is a problem with either the linter configuration or this code.

IMO not a problem to merge the PR with this (unimportant) change but do as you want.

Reply:
I guess both the auto-corrected and the uncorrected format look kind of hideous in the code.